jax/_src/pallas/mosaic_gpu/primitives.py

```diff
@@ -361,6 +364,7 @@ def copy_smem_to_gmem(
     reduction_op: If set, perform the specified reduction operation when storing
       to GMEM. For example, using ``"add"`` is conceptually equivalent to
       doing ``src += dst``.
+    team_id: if set, dst ref would be translated to a multicast memory addr
```

Comment:
How does a Pallas user get control over teams? It's not a JAX-level concept so it doesn't make sense to surface it here. How does XLA manage that?

I think what you can do on the JAX level is take an axis name, and perform the reduction along that JAX mesh axis.

Finally: does it ever make sense to use team_id without reduction_op? If not, we should add checks